app/models/facility_group.rb

```diff
@@ -105,6 +105,10 @@ def syncable_patients
     registered_patients.with_discarded
   end
 
+  def syncable_patient_ids(exclude_facility: nil)
+    syncable_patients.pluck(:id)
+  end
```

Comment:
This is missing the `exclude_facility` exclusion. Let's add a spec for this?

Reply:
Fixed in a0edbc5ce3ed820302f5c2934f74688de9edc445. Will add a spec though.